grails-app/views/_manifest.gsp

```diff
@@ -1,11 +1,12 @@
-<link rel="apple-touch-icon" sizes="180x180" href="http://ala-test.ut.ee/assets/favicons/apple-touch-icon.png?v=eEa25WbRd8">
-<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-32x32.png?v=eEa25WbRd8" sizes="32x32">
-<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-194x194.png?v=eEa25WbRd8" sizes="194x194">
-<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/android-chrome-192x192.png?v=eEa25WbRd8" sizes="192x192">
-<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-16x16.png?v=eEa25WbRd8" sizes="16x16">
-<link rel="manifest" href="http://ala-test.ut.ee/assets/favicons/manifest.json?v=eEa25WbRd8">
-<link rel="mask-icon" href="http://ala-test.ut.ee/assets/favicons/safari-pinned-tab.svg?v=eEa25WbRd8" color="#009FC8">
-<link rel="shortcut icon" href="http://ala-test.ut.ee/assets/favicons/favicon.ico?v=eEa25WbRd8">
-<meta name="msapplication-TileColor" content="#ffffff">
-<meta name="msapplication-TileImage" content="http://ala-test.ut.ee/assets/favicons/mstile-144x144.png?v=eEa25WbRd8">
-<meta name="theme-color" content="#ffffff">
+<link rel="apple-touch-icon" sizes="180x180" href="http://ala-test.ut.ee/assets/favicons/apple-touch-icon.png?v=eEa25WbRd8" />
+<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-32x32.png?v=eEa25WbRd8" sizes="32x32" />
+<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-194x194.png?v=eEa25WbRd8" sizes="194x194" />
+<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/android-chrome-192x192.png?v=eEa25WbRd8" sizes="192x192" />
+<link rel="icon" type="image/png" href="http://ala-test.ut.ee/assets/favicons/favicon-16x16.png?v=eEa25WbRd8" sizes="16x16" />
+<link rel="manifest" href="http://ala-test.ut.ee/assets/favicons/manifest.json?v=eEa25WbRd8" />
+<link rel="mask-icon" href="http://ala-test.ut.ee/assets/favicons/safari-pinned-tab.svg?v=eEa25WbRd8" color="#009FC8" />
+<link rel="shortcut icon" href="http://ala-test.ut.ee/assets/favicons/favicon.ico?v=eEa25WbRd8" />
+
+<meta name="msapplication-TileColor" content="#ffffff" />
+<meta name="msapplication-TileImage" content="http://ala-test.ut.ee/assets/favicons/mstile-144x144.png?v=eEa25WbRd8" />
+<meta name="theme-color" content="#ffffff" />
```

Comment:
Ok, although I think html5 allows closing them like this. Most .gsp files I've seen so far in ALA usually have the closing slash which is why I put them here also